lib/awesome_print/formatter.rb

```diff
@@ -158,7 +158,15 @@ def awesome_object_data(o, variables)
           end
         end
         indented do
-          key << colorize(" = ", :hash) + @inspector.awesome(o.send(var))
+          result = colorize(" = ", :hash)
+
+          if var.to_s.start_with?('@')
+            result += @inspector.awesome(o.instance_variable_get(var))
+          else
+            result += @inspector.awesome(o.send(var))
+          end
+
+          key << result
         end
```

Comment:
How do you feel about the following: 

* Changing the `if` condition to a call, to make it a bit more descriptive and pulling the logic outside of this method?
* We could also write `result += @inspector.awesome(` after the branching to save writing it twice.
* Adding a commit to the send line to explain it's inclusion.

e.g.
```ruby
private

def valid_instance_var?(variable_name)
  variable_name.to_s.start_with?('@')
end

# [...]

  indented do
    var_contents = if valid_instance_var?(var)
      o.instance_variable_get(var)
    else
      o.send(var) # Enables handling of Struct attributes
    end

    key << colorize(" = ", :hash) + @inspector.awesome(var_contents)
  end
```

Thoughts?